src/components/Requirements.vue

```diff
@@ -344,16 +364,44 @@ export default {
 
       return finalRequirementJSONs;
 
+      /**
+       * @typedef {Object} Requirement
+       * @property {string} name
+       * @property {string} description
+       * @property {string} source
+       * @property {string} search
+       * @property {string[][]} includes
```

Comment:
@hanywang2 not entirely sure what the prop includes is for. I saw that below (line 565), there was some explanation but do you think we could put those when listing the properties of the different objects or would that be too verbose?

Reply:
I don't think being too verbose is generally a bad thing for code documentation. `includes` works in conjunction with `search` to scan for courses that satisfy the requirement. For example:
- search = 'code' and includes = 'CS 2110' means the requirement is satisfied when the course code is CS 2110. 
- search = 'subject' and includes = 'INFO' means the requirement is satisfied when the subject is INFO. 
- search = 'catalogSatisfiesReq.' and includes = 'First-Year Writing Seminar.' means the reqyurement is satisfied when the phrase 'First-Year Writing Seminar.' is included in the catalogSatisifesReq attribute of a course's data.

The reason it's called include is because the way it's checked off is by if it's a substring of the course's data (ie if it's included). For example, 'First-Year Writing' can be used instead of 'First-Year Writing Seminar.' to check off the same requirement when search = 'catalogSatisifesReq'.

Reply:
You can read more doc on this in the README.md in src/requirements